src/cockpit/packages.py

```diff
@@ -261,7 +272,7 @@ def load_path(self, path: str, headers: JsonObject) -> Document:
 
         if path == 'po.js':
             locales = parse_accept_language(headers)
-            return self.load_translation(locales)
+            return self.load_translation(self.translations, locales)
```

Comment:
This feels a little bit too implementation-detaily in that Packages reaches a bit too far inside of Package.  What if we handed the `path` to the function and had it decide internally?

Reply:
I'd be up for that but up for that